frontend/src/components/layout/RightColumn/RightColumn.tsx

```diff
@@ -23,7 +23,7 @@ export default component$(() => {
 
 		return (
 			<Link href={linkTarget} class={classList}>
-				<i class={`fa ${iconName} fa-fw md:mr-3`} />
+				<i style={{ width: '20px', height: '16px' }} class={`fa ${iconName} fa-fw md:mr-3`} />
```

Comment:
```suggestion
				<i class={`fa ${iconName} fa-fw md:mr-3 w-5 h-4`} />
```

Reply:
Will this still resolve the CLS issue we're trying to fix? I thought the recommended practice was to set explicit width/height on elements as opposed to waiting for a (potentially bulky css file like tailwind or font-awesome) to download since this will cause a reflow.

https://web.dev/optimize-cls/#images-without-dimensions

Reply:
ah I see, thanks a bunch for the article 🙂 

it says that the best practice is to set the `width` and `height` attributes (not setting them with styles directly), no?

I think css kicks in afterwards, so I think that the best thing would be to do something like:
```suggestion
				<i width="20" height="16" class={`fa ${iconName} fa-fw md:mr-3 w-5 h-4`} />
```
so this would set the initial width and height for the image and then would override it with w-5 and h-4 when the css is ready (w-5 and h-4 are respectively 20px and 16px for the medium font-size so we won't see any difference in that case, but then they will scale properly based on the browser's font size)

alternatively if you prefer you can set directly the sizes in rem directly:
```suggestion
				<i style={{ width: '5rem', height: '4rem' }} class={`fa ${iconName} fa-fw md:mr-3 w-5 h-4`} />
```